loadgen/loadgen.cc

```diff
@@ -746,7 +746,15 @@ PerformanceResult IssueQueries(SystemUnderTest* sut,
       GlobalLogger().GetMaxCompletionTime();
   auto sut_active_duration = max_completion_time - start;
   LogDetail([start_for_power, sut_active_duration](AsyncDetail& detail) {
+#if defined(WIN32) || defined(WIN64)
+    // nanoseconds may lose info when adding to timepoint. msvc will complain about it.
+    // TODO: do we need this on linux to?
+    auto end_for_power =
+        start_for_power +
+        std::chrono::duration_cast<std::chrono::milliseconds::duration>(sut_active_duration);
```

Comment:
Does this work for windows?
std::chrono::duration_cast<std::chrono::system_clock::duration>

If so, let's just switch to that without the ifdef.

Reply:
Huh. My less than greater than signs got ripped out. Hopefully it's clear though.

Reply:
let me try